pkg/operator/csidriveroperator/csioperatorclient/types.go

```diff
@@ -41,6 +41,8 @@ type CSIOperatorConfig struct {
 	OLMOptions *OLMOptions
 	// Run the CSI driver operator only when given FeatureGate is enabled
 	RequireFeatureGate string
+	// Schedule CSI driver operator on workers when control plane is externalized
+	ScheduleOnWorkers bool
```

Comment:
@csrwng on vSphere clusters we also schedule the vsphere-problem-detector pod on master nodes, is that something we need to change as well?

Reply:
Yes, thank you, we should. Will update this PR